src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DataAccess.scala

```diff
@@ -48,4 +48,36 @@ trait DataAccess
       workflowFailureQuery.schema ++
       workflowMessageQuery.schema ++
       workflowAuditStatusQuery.schema
+
+  def truncateAll: WriteAction[Int] = {
+    // important to keep the right order for referential integrity !
```

Comment:
I meant just write a test that calls `truncateAll` and then `SELECT * FROM db.tables_in_me` and hopefully get no results. That way there could be an explicit failure that points to someone having forgotten a table in this function, rather than a maybe-failure depending on which table and whether enough tests hit the same table in the same suite in a way that the lack of truncation would cause a problem.

If killing the unit tests while they're running leaves the db in a nasty state then that's a separate issue that I also think we should address (but maybe not in this PR). I've been able to hook into exit routines to execute custom code in other languages, in the past.